invenio_previewer/webpack.py

```diff
@@ -72,6 +73,9 @@
                 "bottom_css": "./scss/invenio_previewer/bottom.scss",
                 "simple_image_css": "./scss/invenio_previewer/simple_image.scss",
                 "txt_css": "./scss/invenio_previewer/txt.scss",
+                "videojs_js": "./js/invenio_previewer/videojs.js",  # shared for audio and video  # noqa
```

Comment:
I think here you can point directly to the node_modules/video.js module

Reply:
Nice! done.